src/QafooLabs/Bundle/NoFrameworkBundle/Tests/Controller/ResultConverter/ArrayToTemplateResponseConverterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace QafooLabs\Bundle\NoFrameworkBundle\Tests\Controller\ResultConverter;
+
+use PHPUnit\Framework\TestCase;
+
+use QafooLabs\Bundle\NoFrameworkBundle\Controller\ResultConverter\ArrayToTemplateResponseConverter;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use QafooLabs\Bundle\NoFrameworkBundle\View\TemplateGuesser;
+use QafooLabs\MVC\TemplateView;
+
+class ArrayToTemplateResponseConverterTest extends TestCase
```

Comment:
What do you think about marking all tests as `final`? There's probably no need for them to be not `final`.